conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -329,6 +331,12 @@ public void testUnknownContentType() {
                 .hasMessageContaining("INVALID_ARGUMENT");
     }
 
+    @Test
+    public void testSlashesInPathParam() {
+        String expected = "foo/bar/baz/%2F";
+        assertThat(client.path(AuthHeader.valueOf("bearer"), expected)).isEqualTo(expected);
```

Comment:
to test my understanding: prior to this change, this request would not have been matched by the router because undertow would first urldecode the `foo%2Fbar` portion to `foo/bar` and feed that string into the router?

Reply:
Almost, Undertow would decode the URI, but explicitly avoid decoding `%2F` to `/`, and pass `foo%2Fbar` to our handler

Reply:
ok